### ISSUE TITLE
[bug] Microphone audio quality is worse in Windows vs. Chrome

### ISSUE THREAD

**Issue by Unbreathable** [livekit/client-sdk-flutter#576]
**Describe the bug**

Hello, thanks for all your awesome work on LiveKit and making it available for Flutter. Today, I ran into quite the weird issue. Basically when I enter a call on Chrome the audio quality received by the other person sounds much better than when I use the Windows version of the example app provided in this repository. After some testing it seems that this problem mainly affects **mono microphones** since it doesn't happen with my web cam microphone.

**To Reproduce**

1. Export the example app for both Windows and Chrome.
2. Have someone else open either one and join the room.
3. Join the room from Windows and try to speak with a mono (one channel) microphone.
4. Now compare the audio quality to the audio quality you get when speaking through the Chrome version with the exact same microphone.

I've also recorded a video to illustrate the problem (because the video is over 100 MB, I couldn't upload it to GitHub):
https://www.youtube.com/watch?v=ngANWoGaAtA

**Expected behavior**

The audio should sound the same, no matter if I'm in Chrome or Windows.

**Platform information**

* **Flutter version**: 3.24.0
* **Plugin version**: 2.2.4
* **Flutter target OS**: Windows, Chrome
* **Flutter target OS version**: Windows 11

**Comment by cloudwebrtc** [comment]
yeah, it sounds like the sampling rate is too low, will try to track native code.

**Comment by thameraltoimi** [comment]
Facing the same issue as well
